### ISSUE TITLE
Fullscreen colour not accessible to all

### ISSUE THREAD

**Issue by derivadow** [nightscout/nightguard#118]
Colour choice (red green) can't be distinguished if colour blind. 

Better to select accessible colour eg white for normal and red for high

**Comment by dhermanns** [comment]
I think this could be addressed by supporting different color schemes the user is able to switch. I will leave this as a feature request.

**Comment by derivadow** [comment]
I realise this is obviously you're decision but isn't it better to adopt the principles of universal design and design stuff that works for everyone? Less code, more inclusive and more useful for a wider range of people

**Comment by dhermanns** [comment]
I love the new colors. And the green is less bright especially in
fullscreen. So could imagine that this is a benefit for many people.

**Comment by derivadow** [comment]
I agree. I guess I'm trying to make a slightly different point. It is an improvement for many, but why not make it an improvement for all?
